triples-generator/src/main/scala/io/renku/triplesgenerator/events/consumers/membersync/KGSynchronizer.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2022 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.renku.triplesgenerator.events.consumers.membersync
+
+import io.renku.graph.model.projects
+import io.renku.triplesstore.SparqlQuery
+
+private trait KGSynchronizer[F[_]] {
+  def syncMembers(path: projects.Path, membersInGL: Set[GitLabProjectMember]): F[SyncSummary]
+
+  protected def findMembersToAdd(membersInGitLab: Set[GitLabProjectMember],
```

Comment:
Since these two methods don't seem to be related (code-wise) to this trait - what do you think about putting them in a different scope? maybe more utility like one or in some companion? (I suppose once the default graph is removed it we could move it back to the implementation?)